src/restLevelClient/schemaHelpers.ts

```diff
@@ -115,14 +115,17 @@ export function primitiveSchemaToType(
     case SchemaType.Char:
       return "string";
     case SchemaType.ByteArray:
-    case SchemaType.Binary:
     case SchemaType.Duration:
     case SchemaType.Credential:
```

Comment:
Should Credential be mapped to string?

Reply:
That's a goof point. I don't think we have yet any logic for special handling Credentials. We'd need to dig more into that